kolibri/core/auth/api.py

```diff
@@ -621,6 +622,13 @@ def annotate_queryset(self, queryset):
             )
         )
 
+    @decorators.action(methods=["post"], detail=False)
+    def create_facility(self, request):
+        serializer = CreateFacilitySerializer(data=request.data)
```

Comment:
I think the only thing that this is missing is a permissions check here - we should require that `request.user.is_superuser` is `True` otherwise, we should raise a permissions error before doing anything else here.

Reply:
Excellent - yes using the permission classes in the decorator is a very neat way to do this!